docs/en/operation/User-CLI.md

```diff
@@ -823,15 +826,19 @@ Later jobs must wait until some earlier jobs to finish. The default value is `30
 A lower value means slower execution but also being nicer to the other users of the job service.
 * `--overwrite`: Whether to overwrite the destination. Default is true.
 * `--batch-size`: Specifies how many files to be batched into one request. The default value is `20`. Notice that if some task failed in the batched job, the whole batched job would fail with some completed tasks and some failed tasks.
-* `--wait`: true or false specifies whether to wait for command execution to finish. If not explicitly shown then default to wait (true).
+* `--async`: Specifies whether to wait for command execution to finish. If not explicitly shown then default to run synchronously.
 ```console
 $ ./bin/alluxio fs distributedCp --active-jobs 2000 /data/1023 /data/1024
```

Comment:
follow the same format as the console section in distributedCp? at least add an additional example line with `--async` flag

Reply:
follow distributedLoad, yeah, have done and updated